rpc/core/src/server/service.rs

```diff
@@ -46,21 +48,27 @@ use std::{
 /// Subscriber.
 pub struct RpcCoreService {
     consensus: DynConsensus,
+    #[allow(dead_code)] //will find usage
```

Comment:
Write "TODO: Remove this line when utxoindex is connected to an RPC call"

Reply:
Done.